site/package.json

```diff
@@ -15,30 +14,20 @@
     "styled-components": "^3.2.3"
   },
   "scripts": {
+    "dev": "next",
```

Comment:
Is like to keep it as just the “start” script for now.

Reply:
Fixed